popup.js

```diff
@@ -75,6 +75,24 @@ function onFontsLoad(json, fromCache) {
   $('#fonts').on('scroll', throttle(function (event) {
     loadVisibleFonts();
   }, 1000));
+
+  document.getElementById("fonts-search").addEventListener("keyup", onSearchChange);
+}
+
+/**
+ * Apply search to the font list
+ */
+function onSearchChange() {
```

Comment:
Looks pretty good. I usually add debouncing to this sort of thing, but it may not be necessary for such a small amount of DOM. Does the performance seem good?